core/types/gen_receipt_json.go

```diff
@@ -71,8 +71,8 @@ func (r *Receipt) UnmarshalJSON(input []byte) error {
 		TxHash            *common.Hash    `json:"transactionHash" gencodec:"required"`
 		ContractAddress   *common.Address `json:"contractAddress"`
 		GasUsed           *hexutil.Uint64 `json:"gasUsed" gencodec:"required"`
-		EffectiveGasPrice *big.Int        `json:"effectiveGasPrice,omitempty"`
```

Comment:
Please add a comment why `omitempty` is removed

Reply:
I editied description of this PR. The reason why the `omitempty` tag is removed is because the `EffectiveGasPrice` is required field.

Instead of leaving a comment on the code (which is generated by `go generate ...`), I made it into a separate PR so that related changes can be found at any time.